buildbot/master/files/config/master.cfg

```diff
@@ -170,7 +170,8 @@ c['builders'] = [
     DynamicServoBuilder("arm64", CROSS_SLAVES, envs.build_arm64),
     DynamicServoBuilder("linux-dev", LINUX_SLAVES, envs.build_linux),
     DynamicServoBuilder("linux-nightly", LINUX_SLAVES, envs.build_linux),
-    DynamicServoBuilder("linux-rel", LINUX_SLAVES, envs.build_linux),
+    DynamicServoBuilder("linux-rel", LINUX_SLAVES
```

Comment:
It's partially documented right now (the Vagrant-related sections [on the wiki](https://github.com/servo/servo/wiki/Buildbot-administration#making-changes)), but the test suite isn't well documented right now. I have a half written draft sitting on my hard drive somewhere...